frappe/public/js/frappe/misc/datetime.js

```diff
@@ -10,7 +10,7 @@ frappe.provide("frappe.datetime");
 $.extend(frappe.datetime, {
 	str_to_obj: function(d) {
 		// zone hack to remove timezone diff added by momentjs
-		return moment(d, moment.defaultDatetimeFormat).zone(moment().zone())._d;
+		return moment(d, moment.defaultDatetimeFormat)._d;
```

Comment:
@anandpdoshi Will check but it is creating a problem in Chrome, so how to fix the question in Chrome?

I tryied to show it to @rmehta but him ignored, but In my machine I still getting wrong dates